lib/utils/build.js

```diff
@@ -10,7 +10,8 @@ function customPost (program, callback) {
   const directory = program.directory
   let customPostBuild
   try {
-    customPostBuild = require(`${directory}/post-build`)
+    const gatsbyNodeConfig = require(`${directory}/gatsby-node`)
+    customPostBuild = gatsbyNodeConfig.postBuild
   } catch (e) {
     if (e.code !== 'MODULE_NOT_FOUND') {
       console.log('Failed to load custom post build script. It will be skipped.')
```

Comment:
Also the text of the log message should be something like `console.log("Failed to load gatsby-node.js", e)`